new-log-viewer/src/typings/worker.ts

```diff
@@ -41,13 +41,15 @@ enum WORKER_REQ_CODE {
     EXPORT_LOG = "exportLog",
     LOAD_FILE = "loadFile",
     LOAD_PAGE = "loadPage",
+    QUERY_LOG = "queryLog",
```

Comment:
@kirkrodrigues For consistency with the terms in the UI, maybe we should rename "query" -> "search".

Reply:
After having an offline discussion, we decide to use query in the service workers and search in UI.